edison/src/main/java/edu/illinois/cs/cogcomp/edison/features/lrec/WordConjunctionOneTwoThreeGramWindowTwo.java

```diff
@@ -66,10 +63,10 @@ public WordConjunctionOneTwoThreeGramWindowTwo(String viewName) {
             startwin = 0;
         }
 
+        int index = 0;
         for (int i = startwin; i < endwin; i++) {
-
-            window[i] = TOKENS.getConstituentsCoveringSpan(i, i + 1).get(0).getSurfaceForm();
-
+            window[index] = TOKENS.getConstituentsCoveringSpan(i, i + 1).get(0).getSurfaceForm();
```

Comment:
So the size of `window` is `2k+1`, while the number of elements we put inside it is `endwin - startwin = endspan + k - (startspan - k) = 2k + endspan - startspan`, which can be bigger than `2k+1`?